contracts/keep3r-liquidity-manager/Keep3rLiquidityManagerUserLiquidityHandler.sol

```diff
@@ -40,13 +56,42 @@ abstract contract Keep3rLiquidityManagerUserLiquidityHandler is Keep3rLiquidityM
   using SafeMath for uint256;
   using SafeERC20 for IERC20;
 
+  uint256 public constant PRECISION = 1_000;
+  // liquidity fee
+  uint256 public override liquidityFee;
+  // feeReceiver address
+  address public override feeReceiver;
+  // lp => minAmount
+  mapping(address => uint256) public override liquidityMinAmount;
   // lp => amount (helps safely collect extra dust)
   mapping(address => uint256) public override liquidityTotalAmount;
   // user => lp => amount
   mapping(address => mapping(address => uint256)) public override userLiquidityTotalAmount;
   // user => lp => amount
   mapping(address => mapping(address => uint256)) public override userLiquidityIdleAmount;
 
+  constructor(address _keep3rV1) public Keep3rLiquidityManagerParameters(_keep3rV1) {}
+
+  // governor
+  function _setLiquidityFee(uint256 _liquidityFee) internal {
+    // TODO better revert messages
+    require(_liquidityFee <= PRECISION, 'fee-exceeds-100-percent');
```

Comment:
`_liquidityFee <= PRECISION` why ? Shouldn't this be sth like `_liquidityFee <= MIN_LIQUIDITY_FEE` 
and minLiqFee is sth like `1 * PRECISION / 10`. Or maybe just take out precision and have the min liquidity fee and asume 100% precision, dk

Reply:
I'll ad `MIN_LIQUIDITY_FEE` as 10%. makes contract less ruggable